doc/index.rst

```diff
@@ -25,6 +25,24 @@ Using lsst.ts.salobj
     sal_scripts
     :maxdepth: 2
 
+
+Conda Install
+-------------
+ts_salobj is offered as a conda package.
```

Comment:
Note that we also offer salobj in pypi, so you can actually do `pip install ts-salobj` as well.